PlaywrightUltimate/UiTestFixture.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Microsoft.Playwright;
+
+namespace PlaywrightUltimate
+{
+    [Parallelizable(ParallelScope.Self)]
```

Comment:
this file does not do anything useful. It just opens one more empty browser window at the begging of the test and closes it at the end, so it just uses extra resources of your PC